templates/cloudrun.yml

```diff
@@ -104,6 +95,29 @@
         COMMAND_TO_RUN+=( --set-env-vars=${ENV} )
       fi
 
+      if [ -n "${SECRET_YAML}" ]; then
+
+        echo
+        echo "-> ... also deploying kubernetes secrets"
+        echo
+
+        echo "${SECRET_YAML}" > kube/secret.yaml
```

Comment:
Base64 decode is missing. Could we move the temporary file out from the repository path? e.g. /tmp/secret

Reply:
done